CMakeLists.txt

```diff
@@ -95,8 +83,6 @@ foreach(data_file ${data_files_list})
     )
   endif()
 endforeach()
-# Create a test that gets this target
-add_test(NAME get_data_files COMMAND "${CMAKE_COMMAND}" --build "${CMAKE_BINARY_DIR}" --target data_files)
```

Comment:
I think this downloads the data files without having to explicitly run `make`.  Tests that add this as a property ensure that this runs before those tests.